templatetags/htk_tags.py

```diff
@@ -3,6 +3,7 @@
 import urllib
```

Comment:
@ajnassar can you check whether `static/htk/less` was correctly modified or not?